web/web/alerts/static/actions.js

```diff
@@ -15,4 +15,14 @@ export function getAlerts() {
             console.warn(error);
         })
     }
-}
\ No newline at end of file
+}
+
+export function updateAlerts(data) {
+    return dispatch => {
+        api.put('alert', { json: { ...data } }, (response) => {
+            dispatch(getAlerts());
+        }, (error) => {
+            console.warn(error);
```

Comment:
Can we pipe this to the notification component. Kind like the pop up on the login page when a email or password is invalid. Definitely want to start moving away from console logging this type of stuff. We're good for now, but definitely tech debt to clean up later.

Reply:
Sounds good, I think there are a few other parts that need to be cleaned up also. I'll keep track of all that.